src/lib/media/manip.ts

```diff
@@ -203,11 +209,36 @@ function joinPath(a: string, b: string) {
   return a + '/' + b
 }
 
-function normalizePath(str: string, allPlatforms = false): string {
-  if (isAndroid || allPlatforms) {
-    if (!str.startsWith('file://')) {
-      return `file://${str}`
-    }
+function normalizePath(str: string): string {
+  if (!str.startsWith('file://')) {
+    return `file://${str}`
   }
   return str
 }
+
+function createPath(ext: string) {
+  // cacheDirectory will never be null on native, so the null check here is not necessary except for typescript.
+  // we use a web-only function for downloadAndResize on web
+  return `${cacheDirectory ?? ''}/${uuid.v4()}.${ext}`
+}
```

Comment:
This function (and all `expo-file-system` usage) is only used on native, so there are not any situations where `cacheDirectory` would be null.